### ISSUE TITLE
`[nonportable_path_linter] Use file.path() to construct portable file paths.`  possible false positive

### ISSUE THREAD

**Issue by tonyk7440** [r-lib/lintr#1634]
Hello, many thanks for the effort in this package, it is great!

I installed the github version of this package and ran the `lintr:::addin_lint_package()`

The code below is raising the issue `[nonportable_path_linter] Use file.path() to construct portable file paths.`.

I think it must be a false positive?

```
glue("Data from {format(min(model_accuracies$forecast_cycle), '%d/%b/%y')}")
```

**Comment by IndrajeetPatil** [comment]
Same as https://github.com/r-lib/lintr/issues/468

``` r
library(lintr)

lint(
  text = "glue(\"Data from {format(min(model_accuracies$forecast_cycle), '%d/%b/%y')}\")",
  linters = nonportable_path_linter()
)
#> <text>:1:7: warning: [nonportable_path_linter] Use file.path() to construct portable file paths.
#> glue("Data from {format(min(model_accuracies$forecast_cycle), '%d/%b/%y')}")
#>       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

<sup>Created on 2022-10-07 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by IndrajeetPatil** [comment]
Closing in favour of #468 since once that issue is resolved, this issue will also be resolved.
